plugins/joblib_launcher/hydra_plugins/joblib_launcher/joblib_launcher.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from pathlib import Path
+from typing import Any, Dict, Optional, Sequence
+
+from joblib import Parallel, delayed  # type: ignore
+from omegaconf import DictConfig, open_dict
+
+from hydra.core.config_loader import ConfigLoader
+from hydra.core.config_search_path import ConfigSearchPath
+from hydra.core.hydra_config import HydraConfig
+from hydra.core.utils import (
+    JobReturn,
+    JobRuntime,
+    configure_log,
+    filter_overrides,
+    run_job,
+    setup_globals,
+)
+from hydra.plugins.launcher import Launcher
+from hydra.plugins.search_path_plugin import SearchPathPlugin
+from hydra.types import TaskFunction
+
+log = logging.getLogger(__name__)
+
+
+class JoblibLauncherSearchPathPlugin(SearchPathPlugin):
+    """
+    This plugin is allowing configuration files provided by the JoblibLauncher plugin to be discovered
+    and used once the JoblibLauncher plugin is installed
+    """
+
+    def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
+        # Appends the search path for this plugin to the end of the search path
+        search_path.append(
+            "hydra-joblib-launcher", "pkg://hydra_plugins.joblib_launcher.conf",
+        )
+
+
+class JoblibLauncher(Launcher):
+    def __init__(self, joblib: Dict[str, Any] = {},) -> None:
+        """Joblib Launcher
+
+        Launches parallel jobs using Joblib.Parallel. For details, refer to:
+        https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html
+
+        This plugin is based on the idea and inital implementation of @emilemathieutmp:
+        https://github.com/facebookresearch/hydra/issues/357
+        """
+        self.config: Optional[DictConfig] = None
+        self.config_loader: Optional[ConfigLoader] = None
+        self.task_function: Optional[TaskFunction] = None
+
+        self.joblib = joblib
+        for k, v in self.joblib.items():
+            if v == "None":
+                self.joblib[k] = None
```

Comment:
Is there are cleaner way to handle this?

Reply:
See my comment above. (null vs None in YAML).